Forwarded-Upstream: not yet
Author: Raphael Hertzog <hertzog@debian.org>
Comment:
 Since django/bin/profiling/gather_profile_stats.py is installed as an
 executable, it must have a shebang line to really be a working executable.
 .
 It's the only file matching django/bin/*.py that lacks the shebang line.
 .
 Without this patch lintian complains:
 W: python-django: executable-not-elf-or-script ./usr/share/python-support/python-django/django/bin/profiling/gather_profile_stats.py
 .
 The problem has been mentioned upstream in http://code.djangoproject.com/ticket/7268

--- django/bin/profiling/gather_profile_stats.py.orig	2006-12-16 11:15:38.000000000 +0100
+++ django/bin/profiling/gather_profile_stats.py	2006-12-16 11:15:55.000000000 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/python
 """
 gather_profile_stats.py /path/to/dir/of/profiles
 
